libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -176,6 +182,13 @@ public ContentModel(Entities.Content entity) : base(entity)
         this.SourceId = entity.SourceId;
         this.OtherSource = entity.OtherSource;
         this.ProductId = entity.ProductId;
+        this.Product = new ProductModel {
+            Id = entity.ProductId,
+            Name = entity.Product!.Name,
```

Comment:
This will result in null errors.  Handle null values correctly, and also use the constructor when possible to improve usability.
`this.Product = new ProductModel(entity.Product)`

Reply:
Updated, thanks